server/datastore/datastore.go

```diff
@@ -7,9 +7,11 @@ import "time"
 // DB is an interface for the app database.
 // Any Struct used as the app database must implement these functions.
 type DB interface {
-	Init(config DatabaseConfig) error
+	Init(config DatabaseConfig) error // Initialize pgx connection
```

Comment:
Remove the comment. It's a regression (means we fixed it before but we came back).

Reply:
fix: removed comment